packages/api/src/usePaginatedFetch.ts

```diff
@@ -1,47 +0,0 @@
-import { useCallback, useState } from 'react';
```

Comment:
just curious, why are we removing the usage of usePaginatedFetch?

Reply:
@ameerul-deriv,  `usePaginatedFetch` didn't work as expected, when I discussed with @farzin-deriv, when concluded that it'd be best to handle this in the `useP2PAdvertList`, so this had to be removed as we weren't using it anywhere anymore.

Reply:
ahhh got it, thanks for clarifying 🙏